osx_bundle/misc/bundle/launcher.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-bundle=$(cd "$(dirname "$(dirname "$(dirname "$0")")")"; pwd)
+bundle="$(dirname "$(dirname "$(dirname "$0")")")"
```

Comment:
This is needed to make the path absolute in case a relative path gets passed (relative to the bundle root).